InfiniTAM/ITMLib/Engine/DeviceSpecific/CUDA/ITMExternalTracker_CUDA.cu

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014-2015 Isis Innovation Limited and the authors of InfiniTAM
+
+#include "ITMExternalTracker_CUDA.h"
+
+#include <glog/logging.h>
+
+#include "../../DeviceAgnostic/ITMExternalTracker.h"
+#include "../../DeviceAgnostic/ITMPixelUtils.h"
+#include "ITMCUDAUtils.h"
+
+using namespace ITMLib::Engine;
+
+ITMExternalTracker_CUDA::ITMExternalTracker_CUDA(
+    Vector2i imgSize, TrackerIterationType* trackingRegime,
+    int noHierarchyLevels, const ITMLowLevelEngine* lowLevelEngine)
+    : ITMExternalTracker(imgSize, trackingRegime, noHierarchyLevels,
+                         lowLevelEngine, MEMORYDEVICE_CUDA) {
+  LOG(ERROR) << "not implemented yet";
```

Comment:
I removed them here and above since I'm put the function calls for those functions in the right place and by using FATAL, infinitam wont run until I implement them.